docs/user/rstudio/ide/guide/publish/connecting.qmd

```diff
@@ -3,23 +3,29 @@ title: "Connecting"
 date-meta: 2022-12-12
 ---
 
-::: {.callout-warning}
-This section is not complete until we have updated screenshots for publishing to Posit Connect.
-:::
+RStudio provides "Push button deployment" to Posit Connect (formerly RStudio Connect), Shinyapps.io, Posit Cloud (formerly RStudio Cloud), and RPubs.com.
 
-RStudio provides "Push button deployment" to Posit Connect, Shinyapps.io, and RPubs.com. [Posit Connect](https://www.posit.co/products/connect/) supports a spectrum of data products, static or dynamic, developed in R and Python: Dashboards, applications, APIs, reports, and so much more. Connect enables flexible permissions configurations and meets all enterprise security requirements. Shinyapps.io supports hosting of Shiny for R and Python, and RPubs.com supports hosting of static-content only.
+[Posit Connect](https://www.posit.co/products/connect/) supports a spectrum of data products, static or dynamic, developed in R and Python: Dashboards, applications, APIs, reports, and so much more. Connect enables flexible permissions configurations and meets all enterprise security requirements.
 
-Full Documentation for:  
 
-- [Posit Connect User Guide](https://docs.posit.co/connect/user/connecting/)
-- [ShinyApps.io User Guide](https://docs.posit.co/shinyapps.io/)
-- [RPubs Getting Started](https://rpubs.com/about/getting-started)
+[Posit Cloud](https://posit.cloud/)'s publishing features are currently a Beta Feature. The platform supports publishing of static content (Quarto and R Markdown), apps (Shiny for R and Python, Dash, Streamlit) and APIs (Plumber, Flask, FastAPI).
```

Comment:
Maybe we should put Posit Cloud last because it's currently in beta, and then move it higher up when it's in GA? This is minor and unnecessary, though.

Reply:
I thought similar, but Posit Cloud is important, so decided on a beta note and keep it higher.